pylua/network.lua

```diff
@@ -0,0 +1,22 @@
+package.loadlib("libpython2.7.so", "*")
+require('lua-python')
+
+python.execute("import thread")
+python.execute("import time")
+python.execute("import lua")
+python.execute("from pylua.network import *")
+
+WebSocketClient = {}
+
+WebSocketClient.__index = WebSocketClient
+
+function WebSocketClient.WebSocketClient()
+	return python.eval("LuaWebSocketClient()")
+end
+
+function startPythonWebSocketServer()
+	print('start python server')
```

Comment:
@OHerasym, please, expose logger from from python and use it from lua. this is actually looks like a separate task. If it will take much time, then remove all prints from lua code and create new issue for logging from lua via python.
I see the following approach. Add python& lua functions `log_trace`, `log_debug` etc.